evm_loader/cli/src/main.rs

```diff
@@ -704,8 +721,28 @@ fn main() {
             true,
         ).ok();
 
+
+        let rpc_client = if let Some(slot) = app_matches.value_of("slot") {
+            let slot:u64 = slot.parse().unwrap();
```

Comment:
Please handle it properly without unwrap.

Reply:
It is necessary to replace all unwrap() call in the application.
I will do it in next task